test/supp/helgrind-umf_test-disjoint_pool.supp

```diff
@@ -51,3 +51,11 @@
    fun:*glibcxx_rwlock_rdlock*pthread_rwlock_t*
    ...
 }
+
```

Comment:
Probably `pthread_rwlock_{rd,rw}lock` suppressions are not needed anymore. Please, verify.

Reply:
removed